app/javascript/mastodon/features/emoji/unicode_to_unified_name.js

```diff
@@ -0,0 +1,17 @@
+function padLeft(str, num) {
```

Comment:
This is my hunch as to how emoji-mart computes the `unified` name, but again it doesn't work 100% of the time due to inconsistencies between lowercase and uppercase… not sure why.